packages/plugin-select/src/select/select.jsx

```diff
@@ -71,6 +72,19 @@ export default function select(...selectors) {
   };
 }
 
+export default function forwardedSelect(...selectors) {
+  return Component => {
+    const SelectState = select(...selectors)(Component);
+    const forwardRef = (props, ref) => {
+      return <SelectState {...props} _forwardedRef={ref} />;
```

Comment:
Please use forwardedRef instead of _forwardedRef.

Reply:
https://github.com/seznam/IMA.js-plugins/pull/14/commits/af8a31b2ff0a5bf562f56869766f7013e4ee78dc